src/extras/primitives/primitives/a-box.js

```diff
@@ -1,17 +1,15 @@
-var getMeshMixin = require('../getMeshMixin');
 var registerPrimitive = require('../registerPrimitive');
-var utils = require('../../../utils/');
 
-registerPrimitive('a-box', utils.extendDeep({}, getMeshMixin(), {
+registerPrimitive('a-box', {
+  includeMeshProperties: true,
```

Comment:
The flag is too specific ad-hoc to mesh primitives. If you establish the pattern you will have to create other flags for other property groups. I think the previous pattern is less magic and also scale better for future extensions.